apps/mobile/src/navigation/MMBStack.js

```diff
@@ -84,6 +91,12 @@ const StackNavigator: NavigationNavigator<
         title: 'Timeline',
       },
     },
+    [Routes.PASSENGER_DETAIL]: {
```

Comment:
It looks like it's duplicated? Could you please check @KarinaDavtyan ?

Reply:
Sorry just merged😞, going to look into that in next PR